generator/internal/http_option_utils.cc

```diff
@@ -487,15 +486,15 @@ std::string FormatRequestResource(
 // For protos we generate from Discovery Documents the api version is always the
 // last part of the package name. Protos found in googleapis have package names
 // that mirror their directory path, which ends in the api version as well.
-StatusOr<std::string> FormatApiVersionFromPackageName(
+std::string FormatApiVersionFromPackageName(
     google::protobuf::MethodDescriptor const& method) {
   std::vector<std::string> parts =
       absl::StrSplit(method.file()->package(), '.');
   if (absl::StartsWith(parts.back(), "v")) return parts.back();
-  return internal::InvalidArgumentError(
-      absl::StrCat("Unrecognized API version in package name: ",
-                   method.file()->package()),
-      GCP_ERROR_INFO().WithMetadata("method", method.full_name()));
+  GCP_LOG(FATAL) << "Unrecognized API version in package name: "
+                 << method.file()->package()
```

Comment:
Is the filename available?  If so, please log that, and the line number.  That is easier to find than package names.

Reply:
Included the file name.